### PR TITLE
add index.html, CSS, update README, add linters

### PR DESCRIPTION
**Pull request by tiagomarin**
In this push request, I'm adding all the required files and also the linters to make sure my project will pass through Microverse's standard quality control.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,19 @@
...
+> In this project, I will set up a "Hello world" repository to put to practice everything I've learned so far. That includes following the correct Git flow, and using linters when commiting my pieces of code.
+
+## Built With
+
+- Html and CSS
+
+## Authors
+
+👤 **Tiago Lelinski Marin**
+
```
Comment (omarsalem7) [inline README.md:14]: - [ ] [OPTIONAL] It will be better to add your Linkedin account and Twitter if you have one.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello World</title>
+    <link rel="stylesheet" type="text/css" href="styles.css" media="screen" />
+</head>
+<body>
+    <header class="header-color">
+        <h1 >Hello Microverse!</h1>
+    </header>
+</body>
+</html>
```
Comment (tiagomarin) [inline index.html]: I corrected it on another commit. I'm also testing how comments work in here!

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors ✔️
- Descriptive commit messages ✔️

## Required Changes ♻️

- [ ] [Required] Kindly your Pull Request should have a summary describing the introduced changes in more detail.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tiagomarin** [comment]
Thanks for the feedback @omarsalem7 . How do I change it? I've been searching for a way for 15 min and I can just change the title of the PR (when I click on edit). I read that I shouldn't create a new pull request, right?


Actually, I guess I found out. So should I submit it?

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @Tiago-Lelinski-Marin 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**